hooks/useConversations.tsx

```diff
@@ -33,7 +33,14 @@ const useConversations = () => {
       (item: Conversation | ArtistAgent) =>
         'artist_id' in item && item.artist_id === selectedArtist?.account_id
     );
-    return filtered;
+    
+    // Sort by updated_at in descending order (most recent first)
+    return filtered.sort((a, b) => {
+      if ('updated_at' in a && 'updated_at' in b) {
+        return new Date(b.updated_at).getTime() - new Date(a.updated_at).getTime();
+      }
+      return 0;
+    });
```

Comment:
How could we handle ordering at the most upstream point so the data is in the correct order in any part of the app?
- actual: ordering in `hook` useConversations (downstream)
- suggested: ordering in `supabase query` (source) app/api/room/get/route.tsx

My suggestion should result in
- less code changes
- sorted chat across the app (not just useConversations)
![Screenshot 2025-04-03 at 1 02 38 PM](https://github.com/user-attachments/assets/50760992-ad06-44f5-a717-3791dce6fb51)